splice_machine/pom.xml

```diff
@@ -103,6 +104,18 @@
             <groupId>commons-cli</groupId>
             <artifactId>commons-cli</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hbase</groupId>
+            <artifactId>hbase-common</artifactId>
+            <version>1.0.0-cdh5.6.0</version>
```

Comment:
i'm still weirded out by cdh5.6.0 here.

Reply:
It is there because kerberos was integrated directly in the DRDAConnThread I believe.  Probably would make sense to pull Kerberos out into its own package.